gribscan/gribscan.py

```diff
@@ -489,6 +534,7 @@ def build_refs(messages, global_attrs, coords, varinfo, magician):
         refs[info["name"] + "/.zattrs"] = json.dumps(
             {
                 **info["attrs"],
+                **info["extra"],
```

Comment:
This should likely not be in the generic part of `gribscan`, but inside the `Magician`, if needed.
```suggestion
```